docs/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernete's community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Onprem
+- [ ] Controller
+- [ ] Operator
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
+
+- [ ] Local unit-tests
+- [ ] Local Kupe API subsystem tests
+- [ ] assisted-test-infra environment
+- [ ] dev-scripts environment
+- [ ] Reviewer's test appreciated
+- [ ] Waiting for CI to do a full test run
+- [x] No tests needed
+
+It's recommended to take a few extra minutes to provide more information about
+how this code was tested. Here are some questions that may be worth answering:
+
+- Should this PR be tested by the reviewer?
+- Is this PR relying on CI for an e2e test run?
+- Should this PR be tested in a specific environment?
+- Any logs, screenshots, etc that can help with the review process?
+
+
+# Assignees
+
+Please, add one or two reviewers that could help review this PR.
+
+/assign @
+/assign @
+
+## Checklist
+
+- [ ] Title and description added to both, commit and PR.
+- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
+- [ ] Reviewers have been listed
+- [ ] This change does not require a documentation update
```

Comment:
Documentation? Meaning README and such? Godoc/Python Docstring for APIs?

Reply:
All of it, I will be explicit